src/Common/tests/Scripts/Tools/net_providers.txt

```diff
@@ -1,4 +1,5 @@
 "{bdd9a83e-1929-5482-0d73-2fe5e1c0e16d}"
+"{b71555b1-9566-5ce3-27f5-98405bbfde9d}"
```

Comment:
I'm curious how does the unique GUID get generated?

Reply:
I don't know the exact formula.  @vancem can explain it better.  I simply used the code sample above to tell me what the GUID was.

Reply:
Sorry I didn't see the PR description. Yeah that answers my question, thanks!